frontend/src/metabase/common/components/FilterPicker/FilterOperatorPicker.tsx

```diff
@@ -20,5 +20,5 @@ export function FilterOperatorPicker({
     }));
   }, [options]);
 
-  return <Select data={data} {...props} />;
+  return <Select data={data} {...props} data-testid="filter-operator-picker" />;
```

Comment:
WDYT about using an `aria-label` instead? Could be something like "Filter operator"